src/cljs/org/broadinstitute/firecloud_ui/page/workspace/summary/workspace_cloner.cljs

```diff
@@ -59,11 +61,12 @@
      (if-let [fails (input/validate refs "name")]
        (swap! state assoc :validation-error fails)
        (let [name (input/get-text refs "name")
-             project (nth (:billing-projects props) (int (:selected-project @state)))]
+             project (nth (:billing-projects props) (int (:selected-project @state)))
+             protected? (react/call :checked? (@refs "protected-check"))]
          (swap! state assoc :working? true :validation-error nil :error nil)
          (endpoints/call-ajax-orch
            {:endpoint (endpoints/clone-workspace (:workspace-id props))
-            :payload {:namespace project :name name}
+            :payload {:namespace project :name name :attributes {} :isProtected protected?}
```

Comment:
Are we relying on the UI to send the appropriate value for isProtected to the back end? That's not safe - it's trivial to modify the checkbox input using dev tools, re-enable it and uncheck it, then perform the clone. The back end needs to validate the clone value.